src/processed/ProcessedFile.java

```diff
@@ -1,22 +1,84 @@
 package processed;
 
-import java.util.ArrayList;
+import java.io.IOException;
+import java.time.LocalTime;
+import java.util.Date;
+import java.util.HashMap;
 import java.util.List;
+import static java.time.temporal.ChronoUnit.MINUTES;
 
+import lombok.Getter;
+import parser.IParser;
 import row.Row;
 
 public class ProcessedFile {
-	private List<Row> rows;
-	
-	public ProcessedFile() {
-		rows = new ArrayList<>();
+	private IParser parser;
+
+	private HashMap<String, Float> orderHours;
+	private HashMap<Date, Float> dailyHours;
+	private float totalHours;
+	private @Getter List<Row> rows;
+
+	public ProcessedFile(IParser parser) {
+		this.parser = parser;
+		orderHours = new HashMap<String, Float>();
+		dailyHours = new HashMap<Date, Float>();
+		totalHours = 0;
+	}
+
+	public void updateData() throws IOException {
+		rows = parser.parse();
+		for (Row row : rows) {
```

Comment:
@MariusLovesPizza @scrivereleonardo Questo costrutto si chiama **for each** anche questo permette di rendere il codice più pulito e leggero.